workloads/network-perf/compare.py

```diff
@@ -40,14 +43,38 @@
     print("YAML file contains no data")
     exit(1)
 
+def percent_change(value, reference):
+    if reference:
+        return abs(((value-reference)*1.0/reference))*100
+    else:
+        return -1
+
+def compare(data, tolerance, latency=False):
+    ref_val = data[params.uuid[0]]
```

Comment:
does `params.uuid` have scope here? Shouldn't we be passing this as a param to `compare`?